doc/release/platforms/README.cygwin.rst

```diff
@@ -34,14 +34,13 @@ please let us know at chapel_info@cray.com.
 Required Packages
 -----------------
 
-In order to get the required tools listed in README.prereqs, Cygwin
-users will need to ensure that the following packages are installed in
-addition to the standard set of Cygwin packages:
+In order to get the required tools listed in :ref:`readme-prereqs`,
+Cygwin users will need to ensure that the following packages are
+installed in addition to the standard set of Cygwin packages:
 
     Devel
       + gcc and gcc-g++
       + make
-      + patch
```

Comment:
Should we mention the virtualenv symlink we've needed for building the test environment here?